source/extensions/filters/listener/tls_inspector/tls_inspector.h

```diff
@@ -83,6 +90,7 @@ class Filter : public Network::ListenerFilter, Logger::Loggable<Logger::Id::filt
   void done(bool success);
   void onALPN(const unsigned char* data, unsigned int len);
   void onServername(absl::string_view name);
+  bool shouldInspect(uint32_t port) const;
```

Comment:
could we rename this to `shouldInspectPort`?

Reply:
will do